### ISSUE TITLE
Force Two Factor Authentication? 

### ISSUE THREAD

**Issue by woeterman94** [skoruba/IdentityServer4.Admin#734]
Is there a way to force all users to setup Two Factor Authentication?

**Comment by skoruba** [comment]
When do you want to force 2FA? Can you please describe your scenario?

**Comment by woeterman94** [comment]
For extra security. Force all users to set-up two factor authentication when they register or login. (Similar like in outlook, office 365,...)

**Comment by woeterman94** [comment]
Is this something that you would consider adding? We want to be able to force users to setup 2FA.

**Comment by skoruba** [comment]
Yes, I will consider to add it, but in the future. Right now I am working on migration IS4 to version 4. Or if you are interested in send a PR with this feature, you can for sure. 👍🏼
